Tasks/AndroidSigningV3/androidsigning.ts

```diff
@@ -0,0 +1,135 @@
+import * as path from 'path';
+import * as Q from 'q';
+import * as tl from 'vsts-task-lib/task';
+
+const findAndroidTools = (tool: string): string => {
```

Comment:
I think find**In**AndroidTools or findAndroidTool better describes what this function is doing.  Also, would it make more sense to return a `tl.ToolRunner` instead of a `string`?

Reply:
Changed to `findAndroidTool`.